src/app/components/user/AppUserAvatarBubble.vue

```diff
@@ -0,0 +1,148 @@
+<script lang="ts" setup>
+import { computed, PropType, toRefs } from 'vue';
+import AppAvatarFrame from '../../../_common/avatar/AppAvatarFrame.vue';
+import { ComponentProps } from '../../../_common/component-helpers';
+import { Environment } from '../../../_common/environment/environment.service';
+import { ThemeColor } from '../../../_common/theme/variables';
+import AppUserAvatarImg, {
+	assetGuestImage,
+} from '../../../_common/user/user-avatar/AppUserAvatarImg.vue';
+import { User } from '../../../_common/user/user.model';
+import { styleBorderRadiusCircle, styleChangeBg } from '../../../_styles/mixins';
+import { ChatUser } from '../chat/user';
+import AppUserVerifiedWrapper from './AppUserVerifiedWrapper.vue';
+
+const props = defineProps({
+	user: {
+		type: [Object, null] as PropType<User | ChatUser | null>,
+		required: true,
+	},
+	bgColor: {
+		type: String as PropType<ThemeColor>,
+		default: 'bg',
+	},
+	link: {
+		type: String,
+		default: undefined,
+	},
+	disableLink: {
+		type: Boolean,
+	},
+	/**
+	 * Shows verified tick or creator badge when available.
+	 */
+	showVerified: {
+		type: Boolean,
+	},
+	verifiedOffset: {
+		type: Number,
+		default: undefined,
+	},
+	verifiedSize: {
+		type: String as PropType<'big' | 'small' | 'tiny'>,
+		default: undefined,
+	},
+	verifiedPosition: {
+		type: String as PropType<ComponentProps<typeof AppUserVerifiedWrapper>['position']>,
+		default: undefined,
+	},
+	/**
+	 * Shows an avatar frame around the avatar when available.
+	 */
+	showFrame: {
+		type: Boolean,
+	},
+	/**
+	 * Allows extra inset to be added/removed from the avatar frame.
+	 *
+	 * Has no effect if {@link showFrame} is `false`.
+	 */
+	frameInset: {
+		type: Number,
+		validator: value => value === undefined || typeof value === 'number',
+		default: undefined,
+	},
+	/**
+	 * Smooshes it.
+	 */
+	smoosh: {
+		type: Boolean,
+	},
+});
+
+const {
+	user,
+	bgColor,
+	link,
+	disableLink,
+	showVerified,
+	verifiedOffset,
+	verifiedSize,
+	showFrame,
+	frameInset,
+	smoosh,
+} = toRefs(props);
+
+const avatarFrame = computed(() => (isChatUser(user.value) ? undefined : user.value?.avatar_frame));
+
+const maySmooshFrame = computed(
+	() => !isChatUser(user.value) && !!user.value?.avatar_frame && showFrame.value && smoosh.value
+);
+
+const href = computed(() => {
+	if (disableLink.value || !user.value) {
+		return undefined;
+	}
+
+	if (!link?.value) {
+		return Environment.wttfBaseUrl + user.value.url;
+	} else if (link.value === 'dashboard') {
+		return Environment.wttfBaseUrl;
+	} else if (link.value === 'fireside') {
```

Comment:
you don't need this anymore, we shouldn't support "fireside"

Reply:
Should I remove this from `AppUserAvatarImg` as well then?